python/qupled/quantum.py

```diff
@@ -280,3 +280,129 @@ def setGuess(self, fileName : str) -> None:
         guess.matsubara = hdfData["matsubara"]
         self.inputs.guess = guess
         
+class QVSStls(qc.VSStls, Qstls):
+
+    """
+    Class used to setup and solve the quantum VS-STLS scheme as described by
+    `Hayashi and Shimizu <https://journals.jps.jp/doi/pdf/10.1143/JPSJ.48.16>`_ for ground state calculations
+    and by ---Placeholder for the finite temperature qVS paper---.
```

Comment:
Remove this line

Reply:
Not sure how to update it exactly. This is kinda tied to your previous comment. There isn't much literature around qVS.

Reply:
For now I would write something like this: "Class used to setup and solve the quantum VS-STLS scheme". We will add a better description once the qVS paper is out, I just didn't want the "--- placeholder ... " to appear in the documentation